03_Modules/EVDriver/src/module/module.ts

```diff
@@ -264,8 +265,11 @@ export class EVDriverModule extends AbstractModule {
       },
     };
 
-    if (message.payload.idToken.type === OCPP2_0_1.IdTokenEnumType.NoAuthorization) {
-      response.idTokenInfo.status = OCPP2_0_1.AuthorizationStatusEnumType.Accepted;
+    if (
+      message.payload.idToken.type === OCPP2_0_1.IdTokenEnumType.NoAuthorization
+    ) {
+      response.idTokenInfo.status =
+        OCPP2_0_1.AuthorizationStatusEnumType.Accepted;
```

Comment:
Are these format changes due to the pre-commit git action for running prettier?

Reply:
No, those aren't merged yet. I let it format on my local machine using prettier in my IDE. This is currently the correct format as a line not allowed to be longer than 80 char. Take a look at my PR #372 . I suggest moving to 100 chars to have a little less wrapping.